src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -151,6 +157,28 @@ namespace AppInstaller::Utility
         return numGraphemeClusters;
     }
 
+    size_t UTF8ColumnWidth(const NormalizedUTF8<NormalizationC>& input)
+    {
+        ICUBreakIterator itr{ input, UBRK_CHARACTER };
+
+        size_t columnWidth = 0;
+        UChar32 currentCP = 0;
+        int32_t nextBrk = 0;
+
+        currentCP = itr.CurrentCodePoint();
+        nextBrk = itr.Next();
+        while (nextBrk != UBRK_DONE && currentCP != U_SENTINEL)

```

Comment:
>nextBrk [](start = 15, length = 7)

Could drop nextBrk and just use itr.Next() here

Reply:
I blame myself copy pasting code ...

---
In reply to: [488238193](https://github.com/microsoft/winget-cli/pull/562#discussion_r488238193) [](ancestors = 488238193)